examples/server/server.cpp

```diff
@@ -3339,6 +3371,12 @@ struct server_context {
                     }
                 }
 
+                // Update speculative metrics
+                slot.n_draft_accepted += ids.size() - 1; // exclude last sampled token
+                if (slot.n_draft_total > 0) {
+                    slot.draft_accept_ratio = (float)slot.n_draft_accepted / slot.n_draft_total;
+                }
```

Comment:
Should this be moved above in the loop so that `send_final_response()` is correct?

Reply:
the slot.draft_accept_ratio was removed with the redundant var. 

I think slot.n_draft_accepted looks like it is in the right place. I moved it up a bit in the code so it is closer to where `ids` is declared.